post-processor/vsphere/post-processor.go

```diff
@@ -122,6 +139,14 @@ func (p *PostProcessor) PostProcess(ui packer.Ui, artifact packer.Artifact) (pac
 		ovftool_uri += "/Resources/" + p.config.ResourcePool
 	}
 
+	if p.config.ESXiHost != "" {
```

Comment:
I would prefer not to see regexes here. Can we add a flag to manually specify dns host or ip host?

Reply:
I think I'm okay with regexes as long as you add a comment to the regexes giving an example of the kind of string they're supposed to match